### PR TITLE
Fix for dated reminders: issue with none english chars

### PR DESCRIPTION
**Pull request by eyalvo-matrix**
fix for strlen() function does not count correctly none English chars 

image

### CODE REVIEW

File: interface/main/dated_reminders/dated_reminders_add.php [hunk 1]
```diff
@@ -83,7 +83,7 @@
 // ------- check priority, only allow 1-3
     isset($_POST['priority']) and intval($_POST['priority']) <= 3 and
 // ------- check message, only up to 160 characters limited by Db
-    isset($_POST['message']) and strlen($_POST['message']) <= $max_reminder_words and strlen($_POST['message']) > 0 and
+    isset($_POST['message']) and strlen(utf8_decode($_POST['message'])) <= $max_reminder_words and strlen(utf8_decode($_POST['message'])) > 0 and
```
Comment (bradymiller) [inline interface/main/dated_reminders/dated_reminders_add.php]: Should this instead be:
mb_strlen
``````
 isset($_POST['message']) and mb_strlen($_POST['message']) <= $max_reminder_words and mb_strlen($_POST['message']) > 0 and
``````

Comment (bradymiller) [reply interface/main/dated_reminders/dated_reminders_add.php]: Note I am not sure, though. Haven't really used that function before. Interested to hear if that tests well.

Comment (eyalvo-matrix) [reply interface/main/dated_reminders/dated_reminders_add.php]: samething :) but if you wish I can change that to mb_strlen
image

**Comment by sjpadgett** [comment]
For what it's worth, I think I used mb_ functions in custom report to ensure utf8 encoding before sending off for pdf creation with no reported ill effects.

**Pull request merged**